generator/src/main/java/com/scottlogic/deg/generator/generation/field_value_sources/TemporalFieldValueSource.java

```diff
@@ -80,15 +80,12 @@ public long getValueCount() {
 
     @Override
     public Iterable<Object> generateAllValues() {
-
-        if (this.isFinite()) {
-            return () -> new UpCastingIterator<>(
-                    new FilteringIterator<>(new SequentialDateIterator(inclusiveLower, exclusiveUpper),
-                            i -> !blacklist.contains(i)));
-        } else {
-            return generateInterestingValues();
-        }
-
+        return () -> new UpCastingIterator<>(
+            new FilteringIterator<>(
+                new SequentialDateIterator(
+                    inclusiveLower != null ? inclusiveLower : LocalDateTime.MIN,
```

Comment:
Sorry, but this should change to a constant for the min date in ISO-8601 according to our docs

Reply:
See [DataTypes.md](https://github.com/ScottLogic/datahelix/blob/master/docs/DataTypes.md#temporal) and https://en.wikipedia.org/wiki/ISO_8601#Dates

This standard states that < 0000 and > 9999 is permitted if agreed by sender and receiver. For now I think its perfectly acceptable to go with dates >= 0000 and <= 9999 until someone states that they need more.